app-modules/portal/src/Http/Controllers/KnowledgeManagementPortal/KnowledgeManagementPortalArticleController.php

```diff
@@ -47,7 +51,13 @@ class KnowledgeManagementPortalArticleController extends Controller
 {
     public function show(KnowledgeBaseCategory $category, KnowledgeBaseItem $article): JsonResponse
     {
+        if (! Session::exists('guest_id') && ! Auth::guard('contact')->check() && ArticleWasHelpful::active()) {
+            $portalGuest = PortalGuest::create();
+            Session::put('guest_id', $portalGuest->id);
+        }
         $article->increment('portal_view_count');
+        $userType = Session::exists('guest_id') ? 'PortalGuest' : 'Contact';
+        $userid = Session::exists('guest_id') ? Session::get('guest_id') : auth('contact')->user()->id;
```

Comment:
Rather than using the class `Session` let's use the global helper `session()` here and below

Reply:
Done